packages/plugins/RSS3/src/base.ts

```diff
@@ -0,0 +1,18 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { languages } from './locales'
+import { PLUGIN_NAME, PLUGIN_ID, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: '🤔',
+    name: { fallback: PLUGIN_NAME },
```

Comment:
Do you know those fields support i18n? By defining an "i18nKey".

![image](https://user-images.githubusercontent.com/5390719/147312999-6f355c5a-6da6-4ccc-b903-087d3e4f4ab6.png)

Reply:
Now FileService plugin support i18n for those fields. I18n keys used in plugin definition must starts with `__`